app/src/pages/playground/playgroundUtils.ts

```diff
@@ -386,13 +386,25 @@ export function getBaseModelConfigFromAttributes(parsedAttributes: unknown): {
     const provider =
       openInferenceModelProviderToPhoenixModelProvider(data.llm.provider) ||
       getModelProviderFromModelName(data.llm.model_name);
-    const urlInfo = getUrlInfoFromAttributes(parsedAttributes);
+    const { baseUrl } = getUrlInfoFromAttributes(parsedAttributes);
+    const azureConfig =
+      provider === "AZURE_OPENAI"
+        ? getAzureConfigFromAttributes(parsedAttributes)
+        : { deploymentName: null, apiVersion: null, endpoint: null };
+    const modelName =
+      provider === "AZURE_OPENAI" && azureConfig.deploymentName
+        ? azureConfig.deploymentName
+        : data.llm.model_name;
     return {
       modelConfig: {
         ...Object.fromEntries(
-          Object.entries(urlInfo).filter(([_, value]) => value !== null)
+          Object.entries({
+            baseUrl,
+            endpoint: azureConfig.endpoint,
+            apiVersion: azureConfig.apiVersion,
```

Comment:
```suggestion
            endpoint: azureConfig.endpoint,
            apiVersion: azureConfig.apiVersion,
```
rather than tying it specific to azure, why not say it's just a partial of the model config? Could be cleaner down the line

Reply:
I think keeping them conceptually distinct for now is probably better, because the model config itself is currently an amalgamation of different elements: it re-uses `modelName` for deployment name, it has both `baseUrl` and `endpoint`, and it also has `region` just for aws. For more clarity, the model config object should probably be refactored to contain as a discriminated union of provider configs, but will punt on that for now.